packages/p2p/src/constants/modals.js

```diff
@@ -22,14 +32,14 @@ export const modals = {
     FilterModal: React.lazy(() =>
         import(/* webpackChunkName: "filter-modal" */ 'Components/modal-manager/modals/filter-modal')
     ),
+    MyAdsDeleteModal: React.lazy(() =>
```

Comment:
Just re-arranging the constants in alphabetical order and adding 2 new constants: `AddPaymentMethodErrorModal` and `CreateAdAddPaymentMethodModal`